runtime/parachains/src/scheduler/tests.rs

```diff
@@ -116,6 +113,81 @@ fn default_config() -> HostConfiguration<BlockNumber> {
 	}
 }
 
+#[cfg(test)]
```

Comment:
Nit: the whole module is guarded by `#[cfg(test)]` so these here can be removed.